kitsune/sumo/static/sumo/js/ui.js

```diff
@@ -275,6 +276,14 @@
     }
   }
 
+  function correctFixedHeader(){
+    var headerHeight = document.getElementsByTagName("header");
+    var scrollHeight = headerHeight[0].scrollHeight;
+    if(window.location.hash && document.getElementById(window.location.hash)){
```

Comment:
`getElementById` should be `document.querySelector('#' + window.location.hash)`, for the same reason as above.